app/javascript/components/DownloadButton.js

```diff
@@ -29,47 +25,61 @@ function fetchDownloadConfig() {
     'curl -K cfg.txt; rm cfg.txt'
   );
 
-  return [authCode, timeInterval, downloadCommand];
+  return {
+    authCode: authCode,
+    timeInterval: timeInterval,
+    downloadCommand: downloadCommand
+  };
 }
 
 function DownloadCommandContainer() {
-  
-  const [authCode, timeInterval, downloadCommand] = fetchDownloadConfig();
 
-  const expiresMinutes = Math.floor(timeInterval / 60); // seconds -> minutes
+  const [downloadConfig, setDownloadConfig] = useState({});
 
-  const commandID = 'command-' + authCode;
+  useEffect(() => {
+    const fetchData = async () => {
+      const dlConfig = await fetchDownloadConfig();
+      setDownloadConfig(dlConfig);
+    };
+    fetchData();
+  }, []);
+
+  function onSuccess() {
+    console.log('TODO: Show "Copied!" tooltip');
+  }
 
   return (
     <div>
-      <div class="input-group">
+      <div className='input-group'>
         <input
-          id={commandID}
-          class="form-control curl-download-command"
-          value={downloadCommand}
+          id={'command-' + downloadConfig.authCode}
+          className='form-control curl-download-command'
+          value={downloadConfig.downloadCommand}
           readOnly
         />
-        <span class="input-group-btn"> +
+        <span className='input-group-btn'>
+            <Clipboard
+              data-clipboard-target={'#command-' + downloadConfig.authCode}
+              onSuccess={onSuccess}
+              className='btn btn-default btn-copy'
+              data-toggle='tooltip'
+              button-title='Copy to clipboard'
+            >
+              <i className='far fa-copy'></i>
+            </Clipboard>
           <button 
-            id={'copy-button-' + authCode}
-            class="btn btn-default btn-copy" 
-            data-clipboard-target={'#' + commandID}
-            data-toggle="tooltip"
-            title="Copy to clipboard">
-            <i class="far fa-copy"></i>
-          </button>
-          <button 
-            id={'refresh-button-' + authCode}
-            class="btn btn-default btn-refresh glyphicon glyphicon-refresh"
-            data-toggle="tooltip"
-            title="Refresh download command">
+            id={'refresh-button-' + downloadConfig.authCode}
+            className='btn btn-default btn-refresh glyphicon glyphicon-refresh'
+            style={{marginTop: '-3px'}}
+            data-toggle='tooltip'
+            title='Refresh download command'>
           </button>
         </span>
         </div>
```

Comment:
move to external css

Reply:
Addressed in 034ff9d0.  SCP-2144 tracks linting for such issues.